cypress/support/fragments/inventory/inventoryViewSource.js

```diff
@@ -11,5 +13,6 @@ export default {
   },
   waitLoading: () => {
     cy.expect(HTML(including(title)).exists());
-  }
+  },
+  close: () => cy.do(closeButton.click())
```

Comment:
JFYI, you can use _ instead of () in case without function arguments
`_ => cy.do(closeButton.click())`

of course it is not important, but may be interesting